src/test/java/org/ays/common/util/AysRandomUtil.java

```diff
@@ -29,6 +29,17 @@ public static String generatePassword() {
         return FAKER.internet().password();
     }
 
+    public static String generatePasswordWithinRange(int minLength, int maxLength) {
+        if (minLength < 1 || maxLength < minLength) {
+            throw new IllegalArgumentException("Invalid password length range." +
+                    " Ensure minLength >= 1 and maxLength >= minLength.");
+        }
+        if (minLength == maxLength) {
+            return FAKER.internet().password(minLength, minLength + 1).substring(0, minLength);
+        }
+        return FAKER.internet().password(minLength, maxLength);
+    }
```

Comment:
Okunurluluğu arttırmak için boşluklara dikkat edebiliriz. Ek olarak metot adına `generatePassword` denebilir. Java method overloading'e izin verdiği için aynı isimde kullanılabilir. Parametre almadığı zaman parametresiz metot çalışacaktır.
```suggestion
    public static String generatePassword(int minLength, int maxLength) {
    
        if (minLength < 1 || maxLength < minLength) {
            throw new IllegalArgumentException("Invalid password length range." +
                    " Ensure minLength >= 1 and maxLength >= minLength.");
        }
        
        if (minLength == maxLength) {
            return FAKER.internet().password(minLength, minLength + 1).substring(0, minLength);
        }
        
        return FAKER.internet().password(minLength, maxLength);
    }
```

Reply:
@agitrubard overloading bilerek yapmamistim aslinda, metoda deger girdiğimizde o degerlerin ne icin oldukları daha anlasilir olsun diye. Ama düzeltebilirim gerek yok diyorsan ?

Reply:
@htczrrsz Değerlerin isminden ne için oldukları anlaşılıyor aslında o yüzden özel olarak metot adında belirtmemize ihtiyaç yok ✅